candi/cloud-providers/yandex/openapi/cluster_configuration.yaml

```diff
@@ -120,6 +120,13 @@ apiVersions:
                   Yandex Compute Instance disk size in gigabytes.
                 example: 20
                 x-doc-default: 50
+              etcdDiskSizeGb:
+                type: integer
+                description: |
+                  Etcd disk size in gigabytes.
+                example: 10
+                default: 10
+                x-doc-default: 10
```

Comment:
in another providers we do not have x-doc-default

Reply:
Fixed